changelog/V9gQQys8RWiQUxYiQwsQSg.md

```diff
@@ -0,0 +1,2 @@
+level: silent
```

Comment:
why "silent" and no description?

Reply:
"silent: Minor bugfixes where the bugs did not impact many users"